contrib/Makefile.in

```diff
@@ -52,6 +52,9 @@ tidy:
 
 clean: tidy
 	for p in $(plugins); do (cd $$p && $(MAKE) clean); done
+	- if test -e contrib/mpi-proxy-split; then \
+	   cd contrib/mpi-proxy-split && make clean; \
+	  fi
```

Comment:
Right now, we run `make mana` at the top level, and depend on that to build the `mpi-proxy-split` plugin.  That command does several things.  I would worry about adding it to `plugins` here, because we would be trying to build from two different locations.  Also, since this will be a plugin in DMTCP, I believe that the ajority of DMTCP users will not want to automatically build MANA.

Reply:
(And I used to have `contrib/mpi-proxy-split` in the Makefile.in.  That was  a mistake.  Now it reads `mpi-proxy-split`.)